modules/apps/site-initializer/site-initializer-cms/src/main/resources/site-initializer/fragments/group/cms/fragments/sidebar/index.css

```diff
@@ -13,7 +13,7 @@
 	}
 
 	.sidebar-container {
-		border-right: 1px solid #9b92a5;
+		border-right: 1px solid #e7e7ed;
```

Comment:
Would it be useful to have utility classes in Clay like `sidebar-start` and `sidebar-end`  for these border styles?

Reply:
Maybe, but I think we can add that in [this epic](https://liferay.atlassian.net/browse/LPD-11312) and apply it here if we decide to handle styling in that way.